src/api/index.ts

```diff
@@ -11,7 +11,7 @@ const fastify = Fastify({
 fastify.register(mongoConnector);
 fastify.register(routes);
 
-fastify.listen(process.env.PORT || 3000, '0.0.0.0', function (err, address) {
+fastify.listen(process.env.PORT || 3000, "0.0.0.0", function (err, _address) {
```

Comment:
why `_address`?

Reply:
I do this to shut webstorm up about unused parameter, and leave it in for potential future use, better to just remove them?

Reply:
just remove them